lib/infrastructure/models/meeting_model.dart

```diff
@@ -288,8 +288,8 @@ class Meeting extends Equatable {
       energy[k] = data['energy'][k] as int?;
     }
 
-    final DateTime? start = data['start']?.toDate();
-    final DateTime? end = data['end']?.toDate();
+    final DateTime? start = data['start'].toDate();
+    final DateTime? end = data['end'].toDate();
```

Comment:
why removing ? here? we are expecting DateTime?, so we might be expecting no data['start']